tests/Authorization/AuthorizableTest.php

```diff
@@ -295,4 +296,28 @@ public function testCanCascadesToGroupsWithWildcards(): void
 
         $this->assertTrue($this->user->can('admin.access'));
     }
+
+    /**
+     * @see https://github.com/codeigniter4/shield/pull/238
+     */
+    public function testCreatedAtIfDefaultLocaleSetFaWithAddGroup(): void
+    {
+        $currentLocale = Locale::getDefault();
+        Locale::setDefault('fa');
+
+        $this->hasInDatabase('auth_groups_users', [
+            'user_id'    => $this->user->id,
+            'group'      => 'admin',
+            'created_at' => Time::now()->format('Y-m-d H:i:s'),
+        ]);
```

Comment:
`hasInDatabase()` inserts a record:
https://codeigniter4.github.io/CodeIgniter4/testing/database.html?highlight=hasindatabase#hasindatabase-table-data
Why do you need to insert the data here?

Reply:
In fact, I was just trying to check the correct record of the date in the database.
That's why I used it.

Reply:
Oh my God.
This is a ridiculous mistake, **I'm sorry.**
**Thanks for the explanation.**

Reply:
Not your fault @datamweb, I have always felt that `hasInDatabse()` was very poorly named.